### ISSUE TITLE
update lodash dependency

### ISSUE THREAD

**Issue by temsa** [WhoopInc/node-pg-range#7]
```
npm audit
                                                                                
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ pg-range [dev]                                               │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ pg-range > lodash                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 2911 scanned packages
  1 vulnerability requires manual review. See the full report for details.

```

**Comment by ckiss** [comment]
hi @benesch. if I do a PR to fix the above are there any chances that a new version which includes the fix will be published?

**Comment by benesch** [comment]
Sure thing, @ckiss! Thanks for your interest.

**Comment by benesch** [comment]
And it's published as v1.0.1! Enjoy! 🍻

**Comment by ckiss** [comment]
thanks for the update @benesch 👍
